data/human/deep missions.txt

```diff
@@ -3949,6 +3949,7 @@ mission "Stones of Skadenga 1"
 			`	The five Skadenga you brought are polite and circumspect. Upon landing, they hand you <payment>, leave your ship and head for the Goddess-Stone of Skade with its attendant piles of stones. They each navigate their way to a specific cairn, stand there for a moment, speaking or chanting, and then place their stone upon it. Solemnly, they return to you with their task complete. There is nothing else for them to do here, and they request you take them home.`
```

Comment:
The payment in question is in the on offer conversation, and it's stated that it's handed to you upon landing. You've placed the payment in the on complete conversation, where no payment is mentioned. You should instead either hard code this payment to say "50,000 credits," or move the `payment 50000` from the previous mission's on complete into this mission's on offer, and then hard code the `<payment>` replacement from the previous mission to be "50,000 credits."

(I think more correctly this chain's mission structure needs redone, but I'm not interested in diving into that here.)

Reply:
I agree it should probably be redone.  For now, I took the "easy" path and have decided to hard code it.  It comes with the risk that the payment needs to be changed in two places.  But for now, at least it will no longer show up as blank for players.

Reply:
I mean, this is a prime example of when having the conversation in the `on complete` is valid and probably desirable.

Reply:
Yeah, moving the conversation to the previous `on complete` works out the best. That's when you receive the payment. Making Stones of Skadenga 1 a `landing` mission should also be done, since you have to hit the spaceport to receive that text currently, which doesn't seem right. That change would make it seamless.

Reply:
I can take a look at updating this in the evening.

Reply:
How's it now?